### DISCUSSION TITLE
Add preview images to the links

### DISCUSSION THREAD

**Discussion by Uni-versalis** [WebCrateApp/webcrate#31] [Ideas] [answered]
Would it be possible to get small thumbnails plus a button to switch between simple texts (current) to thumbnail mode?

**Comment by BetaHuhn** [comment]
<details>
<summary>For reference</summary>

![image](https://user-images.githubusercontent.com/51766171/132028105-8f8047bf-d605-4a07-93e2-f6d6be427825.png)

</details>

This is something I also have been thinking about. Will add it to the list :)

**Comment by BetaHuhn** [comment, marked as answer]
This is now a thing in [v1.3.0](https://github.com/WebCrateApp/webcrate/releases/tag/v1.3.0)!

I decided to go with full size images instead of a small thumbnail for now. I'm still planning on adding a list mode which will have smaller thumbnails and display the links in a single column list/table.

@Uni-versalis refer to the [docs](https://webcrate.app/docs/faq#how-are-updates-handled) on how you can update to the latest version if you haven't already :)
